Packs/CortexAttackSurfaceManagement/Scripts/GenerateASMReport/GenerateASMReport.py

```diff
@@ -48,402 +50,657 @@ def build_template(args: Dict[str, Any]) -> List[Dict[str, Any]]:
 
-    # Cortex Logo
-    cortex_img = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAcsAAAHKCAYAAABsXmDiAAAACXBIWXMAAFxGAABcRgEUlENBAAAgAElEQVR4nO3dC3BU153n8X/rhQRCIAkkEI8ISwYDEg/zMvYEMHhtB2dtHCceFq+nnIep2nKKsFVOPIknVdRMnGTX2RpCrXeqPHHGGRuK8cYOdhzHdmEPOLPhZZuHeBiCjACLh0BCgBAvqXvrtLql7lZLp7vVj3Pu/X5cslqt27dPnyP61+fcc8/1+Hw+AQAAfcuibgAA6B9hCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgAZhCQCABmEJAIAGYQkAgEYOFRTu9SNvfnVvy6Hqkrxh1cevnKoJ/aVPwn/q9bNPoorcMtqjg3v3htztUT9H2Sbqs0d97t5lDLvl6/M3/ZSz616vv3ThPOp+X/h2Qd7A6wn9Oerz9VWJwfrp3otPSnILzwzJLmgtLyg9E3xstBpS93sCP/lCnsMX8apDn9sX+K+nXMFy9+zLG+UxPSX1dT936Cvy79UXrbZ9IeXqa/s49tX9/OHbh9/u2cYb5bE9j4/2uiLKEdHWPY/vKXvo34A37DkiH9v7+ULv94T9OwnuN3S73mXv9ZwS/vhe9/p6Pyb09XbvM+zfax/7ini8N7IVo/zJh77e/vbmjXhstHrovd/wbaNsvyXwfc+2r6xv7WNXruPp+83J+Xad/mT4p017VzVcOfW1M9dbJh2/3pzf5r3p9r8J6+R5smTCoFIZX1AuxXlFgeITlqHl7v16CcvQx/e6191hGeqizyd7RHwqQDdtX7phT9+bOpsrw3Ljod8+vefCodWHrpwcc/pmmwElQrIMzc6TeUUTpSRvKGFJWBKWUfcbvm1/20vvejwuIi+rr+1LNzT08zDHcU1Yfnpmz/BdZz9de+Bi/YrtbcdyDSgSUqi2YJTcWjhOsrNyCMsojyYswx/f617CsucZI+oxxG9EZI1bQtMVYfn6kTf//t3GP/1w79VTHKN1kdKcAllYXCNZnmzCkrAkLJMflkGuCE1Hh+Xus/tmvHN887tvnv+43IDiIANUYC4orpEcT9fEb8Iy9DZhSVj2vj9SDGGpXPSKrNm5dMNa3Ya2cmxYvnHkrb9/4+TmHx+5ft6A0iCTJg4aIdOHVflLQFiG3iYsCc
```

Comment:
Given that we are instantiating similar templates in the `if` and `elif`, I think that creating a separate function that is called something like `build_template` might be useful here. What do you think?

Reply:
I don't think a separate function helps too much, but let me think about it